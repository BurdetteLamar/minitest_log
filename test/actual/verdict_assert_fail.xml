<log>
  <summary verdicts='2' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert?' type='fail'>
    <section name='Arg: false (FalseClass)'>
      <verdict method='verdict_assert?' outcome='failed' id='0' message='Message 0'>
        <act_value>
          false
        </act_value>
        <exception class='Minitest::Assertion'>
          <message>
            Expected false to be truthy.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Arg:  (NilClass)'>
      <verdict method='verdict_assert?' outcome='failed' id='1' message='Message 1'>
        <act_value>
          nil
        </act_value>
        <exception class='Minitest::Assertion'>
          <message>
            Expected nil to be truthy.
          </message>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
</log>
