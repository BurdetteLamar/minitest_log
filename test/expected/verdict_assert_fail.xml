<log>
  <summary verdicts='2' failures='2' errors='0'/>
  <section name='Test' method='verdict_assert?' type='fail'>
    <section name='Args=[false]'>
      <verdict method='verdict_assert?' outcome='failed' id='0' message='Message 0'>
        <actual class='FalseClass' value='false'/>
        <exception class='Minitest::Assertion' message='Expected false to be truthy.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
    <section name='Args=[nil]'>
      <verdict method='verdict_assert?' outcome='failed' id='1' message='Message 1'>
        <actual class='NilClass' value='nil'/>
        <exception class='Minitest::Assertion' message='Expected nil to be truthy.'>
          <backtrace/>
        </exception>
      </verdict>
    </section>
  </section>
</log>
